src/connection/tests.rs

```diff
@@ -801,7 +805,7 @@ fn from_connect_fetchall_rollback_panic_closed() {
 
 #[test]
 #[serial]
-#[should_panic(expected = "Can't commit while executing")]
+#[should_panic(expected = "Fetching failed: Can't rollback while executing")]
```

Comment:
As this `should_panic` basically tests the result of the `rollback` function, I would get rid of the this `should_panic` and use `assert_eq!`  to check the result.